inst/doc/gsplotIntro.Rmd

```diff
@@ -114,7 +114,7 @@ For timeseries, it is sometimes helpful to plot data on the same graph to make c
 
 ```{r echo=TRUE, fig.cap="Fig. 4 Water temperature timeseries on primary y-axis with pH timeseries on secondary y-axis.", fig.width=6, fig.height=6}
 site <- '04193490'
-demoPlot <- gsplot(mar=c(7.1, 4.1, 4.1, 2.1)) %>% 
+demoPlot <- gsplot(mar=c(7.1, 4.1, 4.1, 4.1)) %>% 
```

Comment:
This plot still shows that if 2 views share an axis, the axis call is being repeated. So, doesn't actual solve #248